src/main/java/io/seqera/tower/cli/shared/WorkflowDumpExportFormat.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2021-2023, Seqera.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.seqera.tower.cli.shared;
+
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.annotation.Nulls;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonAppend;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import io.seqera.tower.JSON;
+import io.seqera.tower.cli.commands.runs.DumpCmd;
+import io.seqera.tower.model.Workflow;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * This is the class used by {@link DumpCmd} to structure Workflow data as JSON.
+ * The class {@link Workflow} is missing some extra fields so this envelope
+ * class is needed to export/import them along the rest of the data.
+ */
+public final class WorkflowDumpExportFormat {
+
+    private Workflow workflow;
+
+    private Long pipelineId;
+
+    private Long workspaceId;
+
+    private Long userId;
+
+    private String userEmail;
+
+    public WorkflowDumpExportFormat(
+            final Workflow workflow,
```

Comment:
I think that it is a bit weird that the parameters being passed are `final` but then the fields of the instance are not. Any reason behind this?

Reply:
A mistake, `final` is missing from the fields.
I'll fix it